ui/admin/app/styles/app.scss

```diff
@@ -1025,13 +1025,29 @@
   text-overflow: ellipsis;
 }
 
-.confirmation-modal {
-  .hds-button-set {
-    margin-bottom: 0;
+.worker-tags {
+  .confirmation-modal {
+    .hds-button-set {
+      margin-bottom: 0;
+    }
+
+    .hds-form-field--layout-vertical {
+      margin-top: 1.5rem;
+      margin-bottom: 0;
+    }
   }
 
-  .hds-form-field--layout-vertical {
+  .edit-modal {
+    .hds-button-set {
+      margin-bottom: 0;
+    }
```

Comment:
How come we needed all this custom css for margins on the hds component? It looks like we're overriding our own override?

Reply:
Indeed with @ZedLi comment, could we give a review to these styles? or explain why we have to add these overrides?